examples/ucp_client_server.c

```diff
@@ -101,35 +101,24 @@ void buffer_free(ucp_dt_iov_t *iov)
     size_t idx;
 
     for (idx = 0; idx < iov_cnt; idx++) {
-        if (iov[idx].buffer == NULL) {
-            continue;
-        }
         mem_type_free(iov[idx].buffer);
-        iov[idx].buffer = NULL;
-        iov[idx].length = 0;
     }
-    mem_type_free(iov);
 }
 
-ucp_dt_iov_t *buffer_malloc()
+ucs_status_t buffer_malloc(ucp_dt_iov_t *iov)
```

Comment:
fill_buffer returns int, buffer_malloc returns ucs_status_t - this is inconsistent
both should return 'int'

Reply:
Done